library/src/lib/time-picker/time-picker.component.ts

```diff
@@ -21,44 +21,64 @@ import { PopperOptions } from 'popper.js';
     styles: [':host {display: inline-block;}']
 })
 export class TimePickerComponent implements ControlValueAccessor, OnInit {
+
+    /** @Input An object that contains three integer properties: 'hour' (ranging from 0 to 23),
+     * 'minute' (ranging from 0 to 59), and 'second' (ranging from 0 to 59).  */
     @Input()
     time: TimeObject = { hour: 0, minute: 0, second: 0 };
 
+    /** @Input Uses compact time picker. */
```

Comment:
Whether the time picker should be in compact mode. maybe?
Uses compact time picker isnt very clear